test/e2e/live_iso_test.go

```diff
@@ -0,0 +1,69 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	bmov1alpha1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("When testing live iso [live-iso]", func() {
+	liveIsoTest()
+})
+
+func liveIsoTest() {
```

Comment:
could you add a log indicating live ISO tests starting to keep it consistent with other tests. You can check this as an example: https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/test/e2e/pivoting_test.go#L33